packages/components/src/components/notification/_inline-notification.scss

```
@@ -48,8 +49,17 @@
   }
 
   .#{$prefix}--inline-notification--low-contrast {
-    color: $text-01;
-    box-shadow: 0 2px 6px 0 rgba(0, 0, 0, 0.2);
+    // Stop-gap to ensure color contrast (vs. fixed background color) until we have component-specific theme tokens
+    color: map-get($carbon--theme--white, 'text-01');
+
+    &:before {
```

can use CSS3 double colon syntax for pseudo elements?

```suggestion
    &::before {
```